src/addin/client-interfaces/addin-modal-config.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Interface for defining configuration options for Modal add-ins
+ */
+export interface AddinModalConfig {
+
+  /**
+   * Indicates that the modal is configured to display full screen (Default: false)
+   *
+   * The appropriate adjustments should be made to the host page when
```

Comment:
This isn't really a valid comment for the client-facing library, right?

Reply:
Yeah, a little too detailed information.  I like the phrasing that I have in the skyapi docs better, so I'll change this to match.